java/client/src/main/java/glide/api/RedisClusterClient.java

```diff
@@ -118,4 +121,36 @@ public CompletableFuture<ClusterValue<String>> info(
         return commandManager.submitNewCommand(
                 Info, options.toArgs(), route, response -> ClusterValue.of(handleObjectResponse(response)));
     }
+
+    /** {@inheritDoc} The command will be sent to a random node. */
```

Comment:
![image](https://github.com/Bit-Quill/glide-for-redis/assets/88679692/63e27f73-01d5-46a9-a00f-ea2960926776)